src/main/java/newladder/model/GameResult.java

```diff
@@ -0,0 +1,24 @@
+package newladder.model;
+
+import java.util.HashMap;
+
+public class GameResult {
+
+    private final HashMap<String, String> resultList = new HashMap<>();
```

Comment:
```suggestion
    private final Map<String, String> resultList = new HashMap<>();
```
[리스코프 치환 원칙] (https://ko.wikipedia.org/wiki/%EB%A6%AC%EC%8A%A4%EC%BD%94%ED%94%84_%EC%B9%98%ED%99%98_ %EC%9B%90%EC%B9%99) 을 만족할 수 있도록 개선해보는건 어떤가요?

Reply:
변수의 이름은 List 인데 실제 타입은 Map을 사용 하고 있네요.